pages/actions/index.js

```diff
@@ -0,0 +1,54 @@
+import getConfig from 'next/config'
+
+// To fetch .env variables from next.config.js
+const {publicRuntimeConfig} = getConfig()
+const {holidayApiURL, holidayApiKey} = publicRuntimeConfig
+
+
+export const SET_COUNTRY = 'SET_COUNTRY'
```

Comment:
all `action_types` should be part of different file

Reply:
created action_types.js